### DISCUSSION TITLE
Stuck on Github Education signup

### DISCUSSION THREAD

**Discussion by shauntneah** [community/community#49378] [GitHub Education]
Hi,

I've registered myself into a bootcamp verified (to Github) academic institution and is trying to sign up for Github Education for use in my bootcamp capstone. 

- my edu email is already verified by GitHub
- the browser location was also allowed
- all fields in the page was filled

However, the signup page is 'stuck' to me after I click on continue. There was no page redirect, no popup, no notification of any kind. The continue button just goes greyed.

what am I doing wrong or is this a github issue?

<img width="774" alt="Screenshot 2023-03-07 at 11 16 33 AM" src="https://user-images.githubusercontent.com/8938302/223312815-77f58754-e849-402d-b06a-e3e52ee2ff62.png">

**Comment by DamaSudo** [comment]
I dont know if were allowed to comment without providing an answer, but I am having this problem too. Thought it was my VPN but after turning it off its still there.

**Reply by AinazRafiei** [reply]
I had the same problem and it's not about VPN. Changing the browser doesn't help either. For me, it was about permissions. Github ask me to access my location and I turn on the location access of my browser and then it worked.

**Comment by samus-aran** [comment]
Hi @shauntneah & @DamaSudo 👋 

Are you still experiencing this? Are you able to test on another browser, Chrome, Edge for example.

Let us know how you get on.

**Reply by SiweiPro** [reply]
I have faced with the same problem, too. Change the browser and even other device cannot work, either.

**Reply by aidarbekq** [reply]
> I have faced with the same problem, too. Change the browser and even other device cannot work, either.

did you find solution?

**Reply by samus-aran** [reply]
Hi @aidarbekq  👋 

Are you able to provide a screenshot of what you're seeing? Do you see any errors on your browser error log via the browser's developer tools?

**Reply by pro402** [reply]
> Hi @aidarbekq 👋
> 
> Are you able to provide a screenshot of what you're seeing? Do you see any errors on your browser error log via the browser's developer tools?

![image](https://github.com/user-attachments/assets/809aab4a-1178-456c-b6dc-c0b27e04c2f4)
![image](https://github.com/user-attachments/assets/4bbe6e9a-5b31-49c2-a904-aaa79c981f1e)
Not able to proceed further how to change the previous details or resubmit the form from start

**Comment by lammchen** [comment]
I had the exact same problem as you and it didn't come from my VPN.

I had been trying to do this on Chromium and failed at it, but whenever I tried it on Firefox it worked.
However, it doesn't seem to work for everyone.

**Comment by chaojiegu** [comment]
Have the same issue.

**Reply by samus-aran** [reply]
Hi @chaojiegu 👋 

Are you able to provide a screenshot of what you're seeing? Do you see any errors on your browser error log via the browser's developer tools?

**Reply by chaojiegu** [reply]
> Hi @chaojiegu 👋
> 
> Are you able to provide a screenshot of what you're seeing? Do you see any errors on your browser error log via the browser's developer tools?

Hi, Thank you for your reply.

I double checked the error log as you suggested. I found that the browser failed to get my location. I swithed to MS edge and tried again and it works.

**Reply by samus-aran** [reply]
Thanks for coming back and replying @chaojiegu 👏 Awesome to hear you were able to proceed! Best of luck 👍

**Reply by aidarbekq** [reply]
> Thanks for coming back and replying @chaojiegu 👏 Awesome to hear you were able to proceed! Best of luck 👍
Hi @samus-aran . I have the same issue. Please help. Errors in log: checked runtime.lastError: The message port closed before a response was received.
DevTools failed to load source map: Could not load content for https://education.github.com/assets/leaflet/dist/leaflet.js-a7a6c324c430b0a049d0a86862b18400583daa4dc10fbe55835cb492baa9fdc1.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

![Снимок экрана (6)](https://user-images.githubusercontent.com/92877850/234851991-c18012c0-ba23-412d-8198-a8f876ed1c1e.png)

**Reply by aidarbekq** [reply]
tried with MS Edge. Still having this issue. MS Edge log errors here 
![Снимок экрана (10)](https://user-images.githubusercontent.com/92877850/234853813-7f64a44a-ce3e-4747-9e6f-f8e404c990a6.png)

**Reply by aidarbekq** [reply]
Also tried on another laptop. No result.

**Reply by aidarbekq** [reply]
@samus-aran

**Comment by yeikel16** [comment]
I have the same isssue hear. The same result, change to browser in my phone and have the same error `To continue you must modify your browser settings to allow GitHub to use your location information.` and in console appears :
```sh
DevTools failed to load source map: Could not load content for https://education.github.com/assets/leaflet/dist/leaflet.js-600a10dc5cd110de0699510d322afcbe01c7ca90b4c5f48adc20314c70aac753.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

**Comment by alirezanobakht13** [comment]
I have the same issue and the reason is `googleapis` is restricted in Iran.

![image](https://github.com/community/community/assets/51335235/bfbb9578-5d5b-4bf4-a70e-e8eebbaabc74)

what should I do?

**Comment by FSWei** [comment]
I am the same. I was supposed to be able to access the authentication interface by the end of July, but today I just clicked 'continue' and got stuck

**Comment by HemboPagi** [comment]
Yep, same issue I have. Tried with different brausers and with and without Uni VPN

**Comment by RickStorage** [comment]
Same issue. Tired different browsers and devices but continues to give the "change your browser settings" error after being blocked the first time by the school location data block.

**Comment by tooltham** [comment]
I faced a similar problem and resolved it by using Google Chrome. To fix the issue, I granted permission to Github to access my location and enabled location access in my browser. This solution worked like magic.

**Comment by HemboPagi** [comment]
Finally i got it to work with Google Chrome. I did not work with Safari an Brave.

**Reply by lfnothias** [reply]
Same here

**Comment by 0x7n6** [comment]
just use chrome or firefox, my safari didnt work

**Comment by nullbyto** [comment]
Works perfectly with Microsoft Edge👍, didn't work with vivaldi or chrome.

**Comment by snaeker58** [comment]
<img width="1440" alt="Screenshot 2024-01-05 at 08 20 52" src="https://github.com/community/community/assets/61626414/234c9b21-4cc8-47c1-b18f-d8c6f2977310">

Still not working, tried Safari, Chrome, Firefox on both Windows, iOS and MacOS...
Can't believe I'm downloading Edge right now.

**Reply by snaeker58** [reply]
<img width="560" alt="Screenshot 2024-01-05 at 08 30 07" src="https://github.com/community/community/assets/61626414/8e5a1bf4-4c44-4365-9e86-102b7d7abfad">
Edge does not work either.

**Reply by snaeker58** [reply]
<img width="553" alt="Screenshot 2024-01-05 at 08 37 47" src="https://github.com/community/community/assets/61626414/aa401988-84a1-4115-9885-05d2fc1e6ce2">
Chrome agrees on not working and has identical errors...

**Reply by MarcoMihalca** [reply]
Same for me.. any fixes...?

**Reply by BriLeeCar** [reply]
Exact same for me as well.

**Comment by jeppeaim** [comment]
I was having the same issue. Enabling location services on my desktop didn't fix it.

What fixed it was opening up the webpage on MS Edge on my phone (s21+), allowing location services. Clicking continue actually worked on my phone. And then I was able to continue the setup on my desktop.

**Reply by MarcoMihalca** [reply]
Nice, for me using my laptop's camera did the trick, even if anything was barely even visible lol.

**Comment by sunset666** [comment]
Just went through the process, my solution. Share my location, I don't know why that is required but clicking the button and allowing it activated the **continue** button.
